tools/spectral/ipa/rulesets/functions/utils/resourceEvaluation.js

```diff
@@ -58,6 +58,9 @@ export function hasGetMethod(pathObject) {
  */
 export function getResourcePaths(parent, allPaths) {
   const childPathPattern = new RegExp(`^${parent}/{[a-zA-Z]+}$`);
-  const customMethodPattern = new RegExp(`^${parent}/{[a-zA-Z]+}:+[a-zA-Z]+$`);
-  return allPaths.filter((p) => parent === p || childPathPattern.test(p) || customMethodPattern.test(p));
+  const customChildMethodPattern = new RegExp(`^${parent}/{[a-zA-Z]+}:+[a-zA-Z]+$`);
+  const customMethodPattern = new RegExp(`^${parent}:+[a-zA-Z]+$`);
+  return allPaths.filter(
+    (p) => parent === p || childPathPattern.test(p) || customMethodPattern.test(p) || customChildMethodPattern.test(p)
```

Comment:
The added tests revealed that we were missing a custom case, as it can be either `/path/{id}:method` or `/path:method`